recipe/meta.yaml

```diff
@@ -132,17 +132,6 @@ outputs:
         - test_data
         - run_test.bat
         - run_test.sh
-      commands:
-        - gdal_grid --version
-        - gdal_rasterize --version
-        - gdal_translate --version
-        - gdaladdo --version
-        - gdalenhance --version
-        - gdalwarp --version
-        - gdalinfo --formats
-        - gdalinfo http://thredds.nersc.no/thredds/dodsC/greenpath/Model/topaz  # [not win]
-        - run_test.bat  # [win]
-        - bash run_test.sh  # [not win]
```

Comment:
@xhochy what is the reason to move these? I have to confess that I prefer them here so we can fix them in a single place rather than in two different files.

Reply:
If there is a `run_test` script, these commands were not executed at all. I moved them into the scripts to be run.

Reply:
Ouch! I had no idea and never checked. That is a bug IMO but let's get this in as-is and maybe we can fix it upstream.